src/testing/postgresql.py

```diff
@@ -99,9 +101,9 @@ def poststart(self):
         with closing(pg8000.connect(**self.dsn(database='postgres'))) as conn:
             conn.autocommit = True
             with closing(conn.cursor()) as cursor:
-                cursor.execute("SELECT COUNT(*) FROM pg_database WHERE datname='test'")
+                cursor.execute("SELECT COUNT(*) FROM pg_database WHERE datname='%s'" % self.settings['database'])
```

Comment:
The database name should be passed as a parameter to execute, instead of string interpolation.